torch/fx/graph.py

```diff
@@ -117,12 +117,6 @@ def _name(self, target: Target) -> str:
         i = self._used_names[op] = self._used_names[op] + 1
         return f'{op}_{i}'
 
-    def get_param(self, name: str) -> Node:
```

Comment:
can we keep these helper functions? or should we always create node through delegate now?

Reply:
Well, it's then easy to mix up delegate vs regular creation. Also, why placeholder and get_param but not others like call_module?

Reply:
> Well, it's then easy to mix up delegate vs regular creation.

so is delegate the suggested API for creating nodes?

> Also, why placeholder and get_param but not others like call_module?

I was planning to add helper functions for other node types as well.

Reply:
Node creation should always go through a `delegate` and the `delegate` should be specifiable by the caller of the pass. Otherwise, we will lose composability, for example if I want to run a pipeline of passes on a Module but I want custom leaf modules

I might go so far as to say we should make `create_node` private, or move it into Delegate itself to make Graph just a dumb container for nodes

Reply:
always enforcing people to use delegate sounds good, are you planning to do it? I still have some prs on quantization, feel free to change all the other call sites of create_node first and I can have a PR to make create_node private and change quantization call sites.